src/network/core/tcp_coordinator.h

```diff
@@ -263,6 +265,28 @@ class NetworkCoordinatorSocketHandler : public NetworkTCPSocketHandler {
 	 */
 	virtual bool Receive_GC_STUN_CONNECT(Packet *p);
 
+	/**
+	 * Game Coordinator informs the client of the NewGRFs in the lookup table.
+	 * This packet is sent after a CLIENT_LISTING reqest, but before GC_LISTING.
+	 *
+	 *  uint32   Listing version.
+	 *  uint16   Number of NewGRFs in the packet, with for each of the NewGRFs:
+	 *  uint32   Lookup table index for the NewGRF.
+	 *  uint32   Unique NewGRF ID.
+	 *  byte[16] MD5 checksum of the NewGRF
+	 *  string   Name of the NewGRF.
+	 *
+	 * The lookup table built using these packets are used by the deserialisation
+	 * of the NewGRFs for servers in the GC_LISTING. These updates are additive,
+	 * i.e. each update will add NewGRFs but never remove them. However, this
+	 * lookup table is specific to the connection with the Game Coordinator, and
+	 * cannot be reused when connecting to another server.
```

Comment:
`and should be considered invalid after disconnecting from the Game Coordinator`. Or at least, `another server` is conflicting in wording, as it is a Game Coordinator, not a Game Server.